table-api/src/main/java/io/deephaven/api/Strings.java

```diff
@@ -59,20 +63,14 @@ public static String of(FilterIn in) {
         return in.toString();
```

Comment:
This seems suboptimal, and worse when inverted with !
I think it should look like something we would parse to a `MatchFilter`. Which means `not in` as an operator when inverted.

Reply:
Note: we aren't currently using Strings for actually printing out anything useful - it's really as helper for label building. When we print out WhereFilter for the engine, it will use the toString of MatchFilter.